cookbooks/bcpc/libraries/utils.rb

```diff
@@ -117,6 +117,16 @@ def get_all_nodes
 	return results.sort
 end
 
+def get_recipe_nodes(recipe_name)
```

Comment:
Can we use [`get_nodes_for`](https://github.com/bloomberg/chef-bach/blob/f1ebce5ef72e189f748b04ef231b396ffe969f0a/cookbooks/bcpc-hadoop/libraries/utils.rb#L125-L132) instead?